easybuild/easyconfigs/g/GC3Pie/GC3Pie-2.2.3.eb

```diff
@@ -0,0 +1,108 @@
+
+exts_list = [
+    ('setuptools', '17.1.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/s/setuptools/'],
+    }),
+    # required for paramiko
+    ('ecdsa', '0.11', {
+        'source_urls': ['https://pypi.python.org/packages/source/e/ecdsa/'],
+    }),
+    # required for paramiko
+    ('pycrypto', '2.6.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/p/pycrypto/'],
+        'modulename': 'Crypto',
+    }),
+    ('paramiko', '1.13.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/p/paramiko/'],
+    }),
+    ('PrettyTable', '0.7.2', {
+        'source_urls': ['https://pypi.python.org/packages/source/P/PrettyTable/'],
+        'source_tmpl': 'prettytable-%(version)s.tar.gz',
+    }),
+    ('pyCLI', '2.0.3', {
+        'source_urls': ['https://pypi.python.org/packages/source/p/pyCLI/'],
+        'modulename': 'cli',
+    }),
+    ('SQLAlchemy', '0.7.9', {
+        'source_urls': ['https://pypi.python.org/packages/source/S/SQLAlchemy/'],
+    }),
+    ('parsedatetime', '0.8.7', {
+        'source_urls': ['https://pypi.python.org/packages/source/p/parsedatetime/'],
+    }),
+    ('boto', '2.9.4', {
+        'source_urls': ['https://pypi.python.org/packages/source/b/boto/'],
+    }),
+    # required for pbr
+    ('pip', '7.0.3', {
+        'source_urls': ['https://pypi.python.org/packages/source/p/pip/'],
+    }),
+    # required for python-novaclient
+    ('pbr', '0.11.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/p/pbr/'],
+    }),
+    # required for python-novaclient
+    ('Babel', '1.3', {
+        'source_urls': ['https://pypi.python.org/packages/source/B/Babel/'],
+    }),
+    # required for python-novaclient
+    ('simplejson', '3.7.3', {
+        'source_urls': ['https://pypi.python.org/packages/source/s/simplejson/'],
+    }),
+    # required for python-novaclient
+    ('requests', '2.4.2', {
+        'source_urls': ['https://pypi.python.org/packages/source/r/requests/'],
+    }),
+    # required for python-novaclient
+    ('iso8601', '0.1.10', {
+        'source_urls': ['https://pypi.python.org/packages/source/i/iso8601/'],
+    }),
+    # required for python-novaclient
+    ('argparse', '1.3.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/a/argparse/'],
+    }),
+    # required for python-novaclient
+    ('six', '1.9.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/s/six/'],
+    }),
+    ('python-novaclient', '2.15.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/p/python-novaclient/'],
+        'modulename': 'novaclient',
+    }),
+    (name.lower(), version, {
+        'source_urls': ['https://pypi.python.org/packages/source/g/gc3pie/'],
+        'modulename': 'gc3libs',
+    }),
+]
+
+pyshortver = '.'.join(SYS_PYTHON_VERSION.split('.')[:2])
+
+# on RHEL-based systems, some extensions get installed to lib, some to lib64
```

Comment:
this relates to https://github.com/hpcugent/easybuild-easyblocks/pull/628

see also http://serverfault.com/a/88739/126446

```
$ ls software/GC3Pie/2.2.3/lib/python2.6/site-packages/
argparse-1.3.0-py2.6.egg  distribute-0.6.14-py2.6.egg  ecdsa-0.11-py2.6.egg-info  novaclient			 pbr			    prettytable-0.7.2-py2.6.egg		     pytz-2015.4-py2.6.egg	  setuptools.pth  six-1.9.0-py2.6.egg
Babel-1.3-py2.6.egg	  easy-install.pth	       gc3pie-2.2.3-py2.6.egg	  paramiko-1.13.0-py2.6.egg	 pbr-0.11.0-py2.6.egg-info  pyCLI-2.0.3-py2.6.egg		     requests-2.4.2-py2.6.egg	  site.py
boto-2.9.4-py2.6.egg	  ecdsa			       iso8601-0.1.10-py2.6.egg   parsedatetime-0.8.7-py2.6.egg  pip-7.0.3-py2.6.egg	    python_novaclient-2.15.0-py2.6.egg-info  setuptools-17.1.1-py2.6.egg  site.pyc

$ ls software/GC3Pie/2.2.3/lib64/python2.6/site-packages/
Crypto	easy-install.pth  pycrypto-2.6.1-py2.6.egg-info  simplejson-3.7.3-py2.6-linux-x86_64.egg  site.py  site.pyc  SQLAlchemy-0.7.9-py2.6-linux-x86_64.egg
```